local-cli/link/ios/copyAssets.js

```diff
@@ -29,7 +29,7 @@ module.exports = function linkAssetsIOS(files, projectConfig) {
     .filter(file => file)   // xcode returns false if file is already there
     .map(file => file.basename);
 
-  plist.UIAppFonts = (plist.UIAppFonts || []).concat(fonts);
+  plist.UIAppFonts = Array.from(new Set([...(plist.UIAppFonts || []), ...fonts]));
```

Comment:
This expression is getting nested a lot and hard to read. Could you please break it into at least two lines, ideally with an intermediate variable with a clear name telling the reader why use the `Set` here (maybe use a name similar to 'deduplicate'?) and ideally also add a comment explaining why it's needed to deduplicate.

Is the `Set` a reliable way to deduplicate the fonts? How does it check equality of two fonts?